<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PregatireOOPTest3Class</class>
 <widget class="QMainWindow" name="PregatireOOPTest3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PregatireOOPTest3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>258</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="carsListWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="FontpushButton">
       <property name="text">
        <string>Show cars with Font</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="carsWithSpecificManufacturerlistWidget"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Manufacturer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="manufacturerlineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="showCarspushButton">
       <property name="text">
        <string>Show cars</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PregatireOOPTest3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FontpushButton</sender>
   <signal>clicked()</signal>
   <receiver>PregatireOOPTest3Class</receiver>
   <slot>ShowCarsWithFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCarspushButton</sender>
   <signal>clicked()</signal>
   <receiver>PregatireOOPTest3Class</receiver>
   <slot>showCarsWithSpecificManufacturer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ShowCarsWithFont()</slot>
  <slot>showCarsWithSpecificManufacturer()</slot>
 </slots>
</ui>
